### DISCUSSION TITLE
attachment [click here to view]

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#4508] [Self hosted] [answered]
Hi, I would like to send attachment but with Mandrill or Sendgrid the email arrive without the attachment and with a link attachment [click here to view] to the active_storage.

How can I send attachment as real attachments?

**Comment by sojan-official** [comment]
We are tracking a fix for this over https://github.com/chatwoot/chatwoot/issues/1063

**Comment by sojan-official** [comment, marked as answer]
fixed in https://github.com/chatwoot/chatwoot/pull/11304
